cats/src/main/scala/magnolify/cats/auto/package.scala

```diff
@@ -16,4 +16,59 @@
 
 package magnolify.cats
 
-package object auto extends LowPriorityImplicits
+import cats._
+import cats.kernel.{Band, CommutativeGroup, CommutativeMonoid, CommutativeSemigroup}
+
+// set implicit priority to avoid conflicts
+// see: https://typelevel.org/cats/guidelines.html#implicit-instance-priority
+// use shapeless.LowPriority so the
+// provided cats type classes are always preferred
+// triggers derivation as last resort
```

Comment:
we unfortunately have to keep `shapeless.LowPriority` to avoid conflicts with implicits provided in `cats._`